content/library/api/connections/connections.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Connections and databases
+slug: /library/api-reference/connections
+---
+
+# Connections and databases
+
+## Setup your connection
+
+<TileContainer>
+<RefCard href="/library/api-reference/connections/st.experimental_connection" size="half">
+
+<Image pure alt="screenshot" src="/images/databases/experimental-connection.png" />
+
+#### Create a connection
+
+Connect to a database and return a connection object.
+
+```python
+st.experimental_connection('pet_db', type='sql')
```

Comment:
we could do a multi-line example if we do no image? (similar to cache data)

```python
conn = st.experimental_connection('pets_db', type='sql')
pet_owners = conn.query('select * from pet_owners')
st.dataframe(pet_owners)
```

Reply:
Fixed on 16e925e